Fix:
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/sockunion.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/prefix.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/if.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/table.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/filter.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/routemap.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/stream.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/plist.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/zclient.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/vrf.o
duplicate symbol '___packed' in:
    .libs/vty.o
    .libs/nexthop.o
https://bugzilla.quagga.net/show_bug.cgi?id=1014
https://bugzilla.quagga.net/attachment.cgi?id=565&action=diff
--- lib/prefix.h.orig
+++ lib/prefix.h
@@ -44,7 +44,7 @@ 
  */
 struct ethaddr {
     u_char octet[ETHER_ADDR_LEN];
-} __packed;
+};
 
 
 /*
